### DISCUSSION TITLE
<MY device laguage>

### DISCUSSION THREAD

**Discussion by Shabihashuchi** [community/community#45006] [Mobile]
### Select Topic Area

Product Feedback

### Body

Hello there.I am new here.I am working on a new generated  device language on my android .At the same time taking a course on 'Machine Learning'  on Coursera  and matlabacademy.mathworks.com .And now I want to flourish my learning in an extra bound.My device language is alphabetic and maths-friendly.I want to go ahead with github;jupyter notebook;lab;binder andbwith other services suitable for me where i can create a friutful coding language and can save all the documents that i will create for a longterm implementations.

**Comment by ghostinhershell** [comment]
Hey there 👋 

Welcome to GitHub and the GitHub Community @Shabihashuchi 🎆 ! We're glad you're here and can't wait to see where your GitHub journey takes you. Check out this [check-in](https://github.com/orgs/community/discussions/42302#discussion-4683140) with links on how to choose your first project to contribute to, a blog with getting started resources, and a new user checklist.

With that being said, you are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this post to the correct category for you. Good luck!

**Reply by Shabihashuchi** [reply]
Thanks.I felt that my github journey will be in an amazing wayHave a lovely day.
